Formula/kubernetes-cli.rb

```diff
@@ -27,7 +27,21 @@ def install
 
     dir = "_output/local/bin/darwin/#{arch}"
     bin.install "#{dir}/kubectl"
+
     (bash_completion/"kubectl").write Utils.popen_read("#{bin}/kubectl completion bash")
+    (zsh_completion/"_kubectl").write Utils.popen_read("#{bin}/kubectl completion zsh")
+  end
+
+  def caveats; <<-EOS.undent
+    Add the following to ~/.zshrc to enable zsh completion:
+      source #{HOMEBREW_PREFIX}/share/zsh/site-functions/_kubectl
+
+    You can configure remote access to Kubernetes clusters with:
+      kubectl config
```

Comment:
This was inspired by the [awscli formula](https://github.com/Homebrew/homebrew-core/blob/master/Formula/awscli.rb). It applies to your previous comment as well.

Now that this is clarified, I can remove the whole caveats block if you wish. Could you confirm?